config/dev.exs

```diff
@@ -116,6 +116,14 @@ config :archethic, ArchEthic.P2P.Listener,
 
 config :archethic, ArchEthicWeb.FaucetController, enabled: true
 
+config :archethic, ArchEthic.Utils.GraphQL.GraphqlServerAPI,
+  client: ArchEthic.Utils.GraphQL.GraphqlClient,
+  query_caller: CommonGraphQLClient.Caller.Http,
+  http_api_url: "http://127.0.0.1:4000/api",
```

Comment:
The issue with this lib, it the host is hard-coded but in the benchmarks the host can change as it's random.
Moreover, it's dependent on nodejs to do schema validation, which is not the best approach.